package.json

```diff
@@ -77,10 +77,10 @@
         "@types/chai": "^4.2.11",
         "@types/chai-as-promised": "^7.1.2",
         "@types/mocha": "^7.0.2",
-        "@types/node": "^13.9.3",
+        "@types/node": "^14.14.8",
         "@types/sinon": "^7.5.2",
-        "@types/square-connect": "^2.20190814.3",
-        "@types/superagent": "^4.1.7",
+        "@types/square-connect": "^4.20201028.0",
```

Comment:
`This Square Connect Node.js SDK is deprecated. The SDK will enter a security maintenance phase in Q4 2020 and will be RETIRED (EOL) in Q2 2021.`

there is a new official nodejs sdk `square@7.x`